packages/source-gdocs-paste/test/atjson-source-gdocs-paste-test.ts

```diff
@@ -0,0 +1,108 @@
+import Document from '@atjson/document';
+import { HIR } from '@atjson/hir';
+import KIXSource from '@atjson/source-gdocs-paste';
+import * as fs from 'fs';
+import * as path from 'path';
+
+describe('@atjson/source-gdocs-paste', () => {
+  describe('relatively complex document', () => {
+    var atjson;
+
+    beforeAll(() => {
+      // https://docs.google.com/document/d/1xP_M2SchJt81ZuivsO7oix8Q_fCx4PENKJFJR5npFNM/edit
+      let fixturePath = path.join(__dirname, 'fixtures', 'complex.json');
+      atjson = JSON.parse(fs.readFileSync(fixturePath));
+    });
+
+    it('has some json', () =>  {
+      expect(atjson).toHaveProperty('resolved')
+    });
+
+    it('does not throw an error when instantiating with KIXSource', () => {
+      expect(new KIXSource(atjson)).toBeTruthy();
+    });
+
+    it('correctly sets the content');
+
+    it('extracts bold', () => {
+      let gdocs = new KIXSource(atjson);
+      let annotations = gdocs.getAnnotations().filter(a => a.type === 'ts_bd');
+      expect(annotations.length).toEqual(2);
+
+      let a0 = annotations[0];
```

Comment:
Very minor, but you could probably destructure these:

```js
let [a0, a1] = annotations;
```

Reply:
👍